Bismuth-Forge/bismuth
# Readme overhaul

## Summary

Added features' demos and overall restructured and overhauled the README.

## Test Plan

1. Open Readme on GitHub
2. Search for typos and grammar errors
3. These should be none
